ts/src/coinone.ts

```diff
@@ -1181,21 +1180,17 @@ export default class coinone extends Exchange {
 
     handleErrors (code: int, reason: string, url: string, method: string, headers: Dict, body: string, response, requestHeaders, requestBody) {
         if (response === undefined) {
-            return undefined;
+            return undefined; // fallback to default error handler
         }
-        if ('result' in response) {
-            const result = response['result'];
-            if (result !== 'success') {
-                //
-                //    {  "errorCode": "405",  "status": "maintenance",  "result": "error"}
-                //
-                const errorCode = this.safeString (response, 'errorCode');
-                const feedback = this.id + ' ' + body;
-                this.throwExactlyMatchedException (this.exceptions, errorCode, feedback);
-                throw new ExchangeError (feedback);
-            }
-        } else {
-            throw new ExchangeError (this.id + ' ' + body);
+        //
+        //     {"result":"error","error_code":"107","error_msg":"Parameter value is wrong"}
```

Comment:
shoildn't we check if the message contains `result:"error"`?

Reply:
The response always contains "result":"success","error_code":"0" or "result":"error","error_code":"ERRORCODE".
Checking result is an unnecessary action, given that error_code is mandatory in any case and is always present

Reply:
got it, thanks